pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
> We could also consider adding a constrained Name field to the signature that is settable by callers. 

If I get this correctly the *constraint* will be a name of containing image as a name's prefix. Followed by some separator (e.g. `#`). And the rest will be up to client's decision:

```go
signature.Name = "sha256:be29438d43e16dc0b63fbf64d7f989e85ccbab9546ac4b23ecd385194d0ff675#rhel7-base"
```

I must say I'd still prefer the `UpdateSignatures()` call over adding some randomly generated (though constrained) name to signature. It seems more like an obstacle to API use.